### PR TITLE
Fixed reset button in oscar template

### LINKED ISSUE #2009: reset button next to search button doesn't work - does nothing
The search button works, but the reset button does nothing. I've tried this in several browsers (Chromium, Chrome, Firefox, & Opera) on both Linux machines and windows 10. Same experience on all - reset button does not work. When I say it doesn't work, it does not clear the search input field.

### PR DESCRIPTION
**Pull request by GazoilKerozen**
Issue #2009

## What does this PR do?

reset button empties the research field in oscar theme as it does in simple 

changed the type "button" and added an Id "clear_search" then created a js function 

## Why is this change important?

because the issue #2009 was open :)

## How to test this PR locally?

1. make a search
2. click the reset "X" button
If the field is empty - it works

## Related issues

If review ok will close #2009

### CODE REVIEW

**Review by return42 — DISMISSED** [review]
Works, thanks!

**Comment by GazoilKerozen** [comment]
Hi @dalf ! Would you like to check out this pull request please ? It'll won't take much time and a review is awaited for our school project (poke @pierrechtux @mrwormo @habsinn @spongebob33 ). I also need a second review to make an update. Thanks a lot ! Cheers

**Comment by dalf** [comment]
I'm sorry about the delay ( https://github.com/searx/searx/issues/2034 need to be fixed )

I'm checking right now. May I ask you:
* rebase the PR with the last 
* run `make theme.oscar` (some commits on the master branch have modified the oscar theme built files).

File: searx/templates/oscar/search.html
Comment (dalf) [inline searx/templates/oscar/search.html]: Can you replace the `active_if_nojs` class with the `hide_if_nojs` class since the button requires Javascript ?

Comment (GazoilKerozen) [reply searx/templates/oscar/search.html]: thanks @dalf ! now I understand better those files in my pull request... ;-) 
I made changes you asked (hope so), it that ok ?

Comment (dalf) [reply searx/templates/oscar/search.html]: I've suggested the change without actually checking, and I have messed up... Sorry.

This issue is when Javascript is disabled (in Firefox:  go to `about:config`, set `javascript.enabled` to False) : the button is still shown but empty.

Comment (dalf) [reply searx/templates/oscar/search.html]: replace line 9 by:
```html
<button type="button" id="clear_search" class="btn btn-default hide_if_nojs" aria-label="{{ _('Clear search') }}">{{ icon('remove') }}</button>
```

tested this time.

[EDIT] the purpose is to hide the `clear` button when javascript is disabled / there is no javascript support.

Comment (GazoilKerozen) [reply searx/templates/oscar/search.html]: that's really kind of you, thanks !

**Comment by dalf** [comment]
Merged.
Thank you !

**Comment by GazoilKerozen** [comment]
my first one ! woohoo :-) 
thak you for help

> Merged.
> Thank you !

**Pull request merged**
